core/common/assets/js/utils/helpers.js

```diff
@@ -1,51 +1,67 @@
+import { softDeprecated, deprecatedMessage } from 'elementor/modules/dev-tools/assets/js/deprecation/utils';
+
 export default class Helpers {
 	softDeprecated( name, version, replacement ) {
-		if ( elementorCommon.config.isDebug ) {
-			this.deprecatedMessage( 'soft', name, version, replacement );
-		}
+		softDeprecated( name, version, replacement );
+
+		// This is is self is deprecated.
+		softDeprecated(
+			'elementorCommon.helpers.softDeprecated',
+			'3.7.0',
+			'elementor.devTools.deprecation.deprecated'
+		);
 	}
 
 	hardDeprecated( name, version, replacement ) {
-		this.deprecatedMessage( 'hard', name, version, replacement );
+		deprecatedMessage( 'hard', name, version, replacement );
+
+		// This is is self is deprecated.
+		softDeprecated(
+			'elementorCommon.helpers.hardDeprecated',
+			'3.7.0',
+			'elementor.devTools.deprecation.deprecated'
+		);
 	}
 
 	deprecatedMessage( type, name, version, replacement ) {
-		let message = `\`${ name }\` is ${ type } deprecated since ${ version }`;
+		deprecatedMessage( type, name, version, replacement );
 
-		if ( replacement ) {
-			message += ` - Use \`${ replacement }\` instead`;
-		}
-
-		this.consoleWarn( message );
+		// This is is self is deprecated.
+		deprecatedMessage(
+			'elementorCommon.helpers.deprecatedMessage',
+			'3.7.0',
+			'elementor.devTools.deprecation.deprecated'
+		);
 	}
 
 	consoleWarn( ...args ) {
-		const style = `font-size: 12px; background-image: url("${ elementorCommon.config.urls.assets }images/logo-icon.png"); background-repeat: no-repeat; background-size: contain;`;
-
-		args.unshift( '%c  %c', style, '' );
+		elementor.devTools.consoleWarn( ...args );
 
-		console.warn( ...args ); // eslint-disable-line no-console
+		// This is is self is deprecated.
+		softDeprecated(
+			'elementorCommon.helpers.consoleWarn',
+			'3.7.0',
+			'elementor.devTools.consoleWarn'
+		);
 	}
 
 	consoleError( message ) {
-		// TODO: function is part of $e.
-		// Show an error if devTools is available.
-		if ( $e.devTools ) {
-			$e.devTools.log.error( message );
-		}
+		// eslint-disable-next-line no-console
+		console.error( message );
```

Comment:
Why it's deprecated?

Reply:
See the old code, the method existed only for handling Web CLI HookBreak and when to console error. 

There are no real requirements for `consoleError`.

And ofc it should not be part of 'core/common/assets/js/utils/helpers.js'.